<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileEncryptionDialog</class>
 <widget class="QDialog" name="FileEncryptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Encryption</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="5" column="0">
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Command</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="uiStart">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Files</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="2">
         <widget class="QPushButton" name="uiInputBrowse">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="uiOutputBrowse">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Output File:</string>
          </property>
          <property name="buddy">
           <cstring>uiOutputFile</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Input File:</string>
          </property>
          <property name="buddy">
           <cstring>uiInputFile</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="uiOutputFile"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="uiInputFile"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Key Path</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Key File Path:</string>
          </property>
          <property name="buddy">
           <cstring>uiKeyFilePath</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="uiKeyFilePath"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="uiSelectKeyFile">
          <property name="text">
           <string>Select...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Passphrase</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Password:</string>
          </property>
          <property name="buddy">
           <cstring>uiPassword</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Confirm:</string>
          </property>
          <property name="buddy">
           <cstring>uiConfirmPassword</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="uiPassword">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="uiConfirmPassword">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="title">
        <string>Parameters</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="3">
         <widget class="QPushButton" name="uiParameters">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Parameters...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Encryption:</string>
          </property>
          <property name="buddy">
           <cstring>uiEncryptRadio</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="uiEncryptRadio">
          <property name="text">
           <string>Encrypt</string>
          </property>
          <property name="icon">
           <iconset resource="EncryptPad.qrc">
            <normaloff>:/images/famfamfam/page_white_key.png</normaloff>:/images/famfamfam/page_white_key.png</iconset>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="uiDecryptRadio">
          <property name="text">
           <string>Decrypt</string>
          </property>
          <property name="icon">
           <iconset resource="EncryptPad.qrc">
            <normaloff>:/images/famfamfam/page_white_text.png</normaloff>:/images/famfamfam/page_white_text.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionSwitchDirection">
   <property name="text">
    <string>SwitchDirection</string>
   </property>
  </action>
  <action name="actionIOChange">
   <property name="text">
    <string>IOChange</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>uiEncryptRadio</tabstop>
  <tabstop>uiDecryptRadio</tabstop>
  <tabstop>uiParameters</tabstop>
  <tabstop>uiInputFile</tabstop>
  <tabstop>uiInputBrowse</tabstop>
  <tabstop>uiOutputFile</tabstop>
  <tabstop>uiOutputBrowse</tabstop>
  <tabstop>uiPassword</tabstop>
  <tabstop>uiConfirmPassword</tabstop>
  <tabstop>uiKeyFilePath</tabstop>
  <tabstop>uiSelectKeyFile</tabstop>
  <tabstop>uiStart</tabstop>
 </tabstops>
 <resources>
  <include location="EncryptPad.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FileEncryptionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FileEncryptionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiEncryptRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>uiParameters</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiEncryptRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>uiConfirmPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiEncryptRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>uiPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiEncryptRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionSwitchDirection</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiInputFile</sender>
   <signal>textChanged(QString)</signal>
   <receiver>actionIOChange</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiOutputFile</sender>
   <signal>textChanged(QString)</signal>
   <receiver>actionIOChange</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
